ansible/roles/operator-pipeline/templates/openshift/tasks/evaluate-preflight-result.yml

```diff
@@ -53,7 +53,12 @@ spec:
       workingDir: $(workspaces.input.path)
       image: "$(params.pipeline_image)"
       env:
-        - name: GITHUB_TOKEN
+        - name: GITHUB_TOKEN # Token used by internal CLI tool
+          valueFrom:
+            secretKeyRef:
+              name: $(params.github_token_secret_name)
+              key: $(params.github_token_secret_key)
+        - name: GH_TOKEN # Token used by the GH tool
```

Comment:
Is this really needed? According to the [docs](https://cli.github.com/manual/gh_help_environment), the `gh` tool should work with either `GITHUB_TOKEN` or `GH_TOKEN`.

Reply:
Ah, good point. I removed the this section.